src/Verify.Tests/Serialization/SerializationTests.ShouldScrubInlineGuidsByDefault.verified.txt

```diff
@@ -0,0 +1,6 @@
+{
+  Title: item Guid_1 - (ID={Guid_1}),
+  Variant: {
+    Id: variant id: Guid_2
```

Comment:
why have the single quotes around the text gone?

Reply:
Good catch. Thing is that up the stack there is a code that does this:

```
if (sharedScrubber.TryConvertString(valueAsString, out var result))
{
    writer.WriteRawValue(result);
    return;
}
```

The `WriteRawValues` expects you to have quotes in place. Changing it to `WriteValues` puts the quotes for all the values. I'll send an update that'd put in the quotes manually when formatting the string.